docs/api-gateway/08-01-expose-service.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Expose a service
+type: Tutorials
+---
+
+This tutorial shows how to expose service endpoints and configure different allowed HTTP methods for them using the API Gateway Controller.
+
+The tutorial comes with a sample HttpBin service deployment.
+
+## Deploy and expose a service
+
+Follow the instruction to deploy an unsecured instance of the HttpBin service and expose it.
+
+1. Export this value as an environment variable:
+
```

Comment:
For the sake of better rendering, I would suggest adding an indent (tab should be enough) in front of code snippets, notes, and basically everything else under numbered lists. You can see the difference it makes below. We basically try to do that with all docs that we render on the website.
![image](https://user-images.githubusercontent.com/40655785/96975898-6f449d00-151b-11eb-92d4-c13735ec62cd.png)